zulipterminal/core.py

```diff
@@ -115,6 +115,10 @@ def show_msg_info(self, msg: Any) -> None:
         self.show_pop_up(msg_info_view,
                          "Message Information (up/down scrolls)")
 
+    def show_stream_info(self, color: str, name: str, desc: str) -> None:
+        show_stream_view = StreamInfoView(self, color, name, desc)
+        self.show_pop_up(show_stream_view, "#{}".format(name))
```

Comment:
We could do with a space here?

Without color, I think the 'P' form we use for private streams would be confusing, so this is fine for now. If we can figure out color then that could be changed later.